examples/provisioner/bottlerocket.yaml

```diff
@@ -19,3 +19,15 @@ spec:
     karpenter.sh/discovery: "${CLUSTER_NAME}" # replace with your cluster name
   securityGroupSelector:
     karpenter.sh/discovery: "${CLUSTER_NAME}" # replace with your cluster name
+  blockDeviceMappings:
+    - deviceName: /dev/xvda
+      ebs:
+        volumeType: gp3
+        volumeSize: 2Gi
+        deleteOnTermination: true
+    - deviceName: /dev/xvdb
+      ebs:
+        volumeType: gp3
+        volumeSize: ${volumeSize} # replace with your required disk size
+        deleteOnTermination: true
+  tags: ${tags}
```

Comment:
To keep things simple to run these, can you remove this? Right now the only env var to replace is CLUSTER_NAME which makes instructions easy to run `envsubt -v '$CLUSTER_NAME' < examples/provisioner/bottlerocket.yaml`

```suggestion
```